framework/src/main/java/org/checkerframework/framework/type/StructuralEqualityVisitHistory.java

```diff
@@ -32,19 +32,25 @@ public StructuralEqualityVisitHistory() {
     /**
      * Add result of comparing {@code type1} and {@code type2} for structural equality for the given
```

Comment:
Like in SubtypeVisitHistory, change the `Add` to `Put` to be consistent with the new method name?